doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+    _For all tables in the database:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
+    ```
+
+    The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
+
+    We strongly recommend that you **limit** publications only to the tables you need.
+
+1. Create a user for Materialize:
+
+    ``` sql
+    CREATE USER materialize PASSWORD '<password>';
+    ```
+
+1. Grant the user permission to manage replication:
+
+    ``` sql
+    GRANT rds_replication TO materialize;
+    ```
+
+1. Grant the user the required permissions on the tables you want to replicate:
+
+    ```sql
+    GRANT CONNECT ON DATABASE <dbname> TO materialize;
+    ```
+
+    ```sql
+    GRANT USAGE ON SCHEMA <schema> TO materialize;
+    ```
+
+    ```sql
+    GRANT SELECT ON ALL TABLES IN SCHEMA <schema> TO materialize;
+    ```
+
+    **Note:** `SELECT` privileges on the tables you want to replicate are needed for the initial snapshot.
+
+## Step 3. Configure your network
+
+When your PostgreSQL database is running in a private network, there are various ways to allow Materialize to connect:
+
+- **Allow Materialize IPs:** This is the simplest approach. You configure your database's security group to allow public connections from a set of static Materialize IP addresses and rely on PostgreSQL authentication and authorization to keep your data safe.
+
+- **Use AWS PrivateLink** or **Use an SSH tunnel:** If you need an additional layer of security at the network level, you can use either [AWS PrivateLink](https://aws.amazon.com/privatelink/) or an SSH tunnel to give Materialize secure access to your database.
+
+Select the option that works best for you.
+
+{{< tabs >}}
+
+{{< tab "Allow Materialize IPs">}}
+
+1. Connect `psql` to Materialize.
+
+    You can find your `psql` connection command on the **Connect** screen in the [Materialize UI](https://console.materialize.com/).
+
+1. Find your static egress IP addresses:
+
+    ```sql
+    SELECT * FROM mz_egress_ips;
+    ```
+1. In the RDS Console, [add an inbound rule](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule) to your database's security group:
+
+    Type | Protocol | Port range | Source
+    -----|----------|------------|-------
+    Custom TCP | TCP | `5432` | IP address from the previous step
```

Comment:
I think we should recommend the PostgreSQL Type, which will auto-set the other fields:

 | Type | Protocol | Port range | Source
 | ---|---|---|---
 |  PostgreSQL | TCP | `5432` | IP address from the previous step as a CIDR


--
>In the RDS Console, [add an inbound rule](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule)

At time of writing there will be 4 IPs that they need to add inbound security rules for.

Might also be worth noting that the AWS console will auto-convert the IPs to CIDRs which is what the security rule actually needs. If they're doing this some other way they may need to do that themselves.

Reply:
nit: I'd inverted this table so its columns are its rows:

```
Field | Value
------|-------
Type  | PostgreSQL
...
```